ps_viewedproduct.php

```diff
@@ -54,36 +56,36 @@ public function __construct()
         parent::__construct();
 
         $this->displayName = $this->trans('Viewed products block', array(), 'Modules.Viewedproduct.Admin');
-        $this->description = $this->trans('Adds a block displaying recently viewed products.', array(), 'Modules.Viewedproduct.Admin');
+        $this->description = $this->trans(
+            'Adds a block displaying recently viewed products.',
+            array(),
+            'Modules.Viewedproduct.Admin'
+        );
 
 
-        $this->templateFile = 'module:ps_viewedproduct/views/template/hook/ps_viewedproduct.tpl';
+        $this->templateFile = 'module:ps_viewedproduct/views/templates/hook/ps_viewedproduct.tpl';
     }
 
     public function install()
     {
         return parent::install()
             && Configuration::updateValue('PRODUCTS_VIEWED_NBR', 8)
             && $this->registerHook('displayFooterProduct')
-            && $this->registerHook('displayProductButtons')
+            && $this->registerHook('displayProductButtons') //Former version of displayProductAdditionalInfo
+            && $this->registerHook('displayProductAdditionalInfo')
```

Comment:
This method is only called for merchant installing the module.

For the others, you need to handle the upgrade which register the hook on the new version: https://devdocs.prestashop.com/1.7/modules/creation/enabling-auto-update/

Reply:
done ^^